### ISSUE TITLE
Index Atala PRISM metadata...

### ISSUE THREAD

**Issue by mgajda** [input-output-hk/cardano-db-sync#1316]
Atala PRISM DID actions are encoded as protocol buffers on the Cardano blockchain.

Because of this they are not indexed by any of the current indexers.

It would be nice to recognize all Atala PRISM action metadata, unpack it and put into Atala table.
This would allow Atala users to explore certificate chains directly from Cardano-DB-sync.

**Comment by mgajda** [comment]
Yes, they are protocol buffers in format specific for the Atala PRISM project.
At the moment, there is no direct way to access these with a smart contract, but might be possible to do it indirectly.

**Comment by profd2004** [comment]
Closing this as stale as part of a triage pass over the older open issues — Atala PRISM metadata indexing was never picked up.

Feel free to reopen if there's still demand.
